packages/runtime/container-runtime/src/deltaScheduler.ts

```diff
@@ -117,6 +132,8 @@ export class DeltaScheduler {
                 numberOfOps: this.schedulingLog.numberOfOps,
                 numberOfTurns: this.schedulingLog.numberOfTurns,
                 processingTime: this.schedulingLog.totalProcessingTime,
```

Comment:
If I read code correctly, processingTime is the time we spent processing ops (minus waiting due to timeouts injected by this code, but not including waiting due to unrelated code pausing op processing). It would be great to capture also total time from start of measurement till end of measurement, which would including amount of delays we injected here.

Reply:
The processingTime during InboundOpsProcessingTime counts for everything. 
Just so I can understand, for this particular event, do you want to include the time it took from pausing until resume gets called?

Reply:
I am a little unclear here. My understanding is that there are 2 kind of delays this code ends up introducing:
1. Time taken to pause the queue and then resume it.
2. Time spend by other code that uses the JS thread when this code releases it.
Do you mean to say we measure both these times?

Reply:
My goal was to collect the elapsedTime (before we pause)  and then , before we resume, when the timeout lambda gets called. But I'm not sure if it will be useful - while debugging it seems to be :)